packages/edgeidentity/js/index.d.ts

```diff
@@ -13,4 +13,27 @@ governing permissions and limitations under the License.
 export class AEPIdentity{
     static extensionVersion(): Promise<string>;
     static getExperienceCloudId(): Promise<string>;
-};
\ No newline at end of file
+    static getIdentities(): Promise<IdentityMap>;
```

Comment:
These APIs seem to require more work, will you add that in the next PR?

Reply:
I will have more code in next PR.  Let review them there.